ports/rp2/mpconfigport.h

```diff
@@ -158,6 +158,8 @@
 // Number 3 is currently used by pico-sdk (PICO_TIME_DEFAULT_ALARM_POOL_HARDWARE_ALARM_NUM)
 #define MICROPY_HW_SOFT_TIMER_ALARM_NUM         (2)
 
+#define MICROPY_STACK_CHECK_MARGIN              (256)
```

Comment:
Please move this config option to where it is relative to how it's ordered in `py/mpconfig.h`.

That would mean placing it between `MICROPY_ENABLE_GC` and `MICROPY_ENABLE_EMERGENCY_EXCEPTION_BUF`.

Reply:
Done. Do you want the same thing for esp32 mpconfigport.h? It would mean splitting up a block of defines with an ifdef.

Reply:
Yes, please move it.  Consistency is better here.

Reply:
Done!